test/lib/test-init.ts

```diff
@@ -0,0 +1,43 @@
+import { fork } from 'child_process';
+import { exit } from 'process';
+export const testListenPort = 1337;
+export const testLocalServer = `http://localhost:${testListenPort}`;
+
+export async function testInit(
+	fixturePath: string,
+	deleteDb: boolean = false,
+	pineListenPort: number = testListenPort,
+) {
+	let testServer: any;
+	try {
+		const processArgs = {
+			fixturePath,
+			testListenPort: pineListenPort,
+			deleteDb: deleteDb || process.env.DELETE_DB,
+		};
+
+		testServer = fork(
```

Comment:
`testServer` is only used within this `try` so you can declare it directly ehre instead and also avoid the `any`
```suggestion
		const testServer = fork(
```

Reply:
thx, done